### ISSUE TITLE
New Version available link points to wrong url

### ISSUE THREAD

**Issue by miyconst** [Starcounter/Starcounter#2977]
At the administrator at `localhost:8181` I see a link 
```
New Version available (v2.1.233)
```

which points to
```
http://127.0.0.1:8181/download/Starcounter/DevelopNightlyBuilds/2.1.233
```

The link should either be fixed or removed.

**Comment by andwah** [comment]
@miyconst Can you please check which version of Starcounter you have installed in the [Administrator Settings](http://localhost:8181/#/server/settings)
![image](https://cloud.githubusercontent.com/assets/2656996/10634619/70df9806-77f3-11e5-8a62-08c39119e565.png)

**Comment by miyconst** [comment]
It was `2.1.143`.

**Comment by andwah** [comment]
what did it say before the version number. 
 See previous picture (**Starcounter-CustomBuilds-2.0.0**) ?

**Comment by miyconst** [comment]
`Starcounter-DevelopNightlyBuilds-2.1.143`

**Comment by andwah** [comment]
thanks..  i will take a look

**Comment by andwah** [comment]
Found the issue, reolved and fixed 19819d5f7f8813a311002a364e78814ceb38831e
